### PR TITLE
Created my first Goodie.

### PR DESCRIPTION
**Pull request by sagarhani**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
If we type duckduckhack in duckduckgo search engine it will show "sagarhani is awesome and has completed the Goodie tutorial!" 

**What problem does your Instant Answer solve (Why is it better than organic links)?**
This is just a basick goodie to show the world the "sagarhani is awesome"

**What is the data source for your Instant Answer? (Provide a link if possible)**

**Why did you choose this data source?**
common

**Are there any other alternative (better) data sources?**
yes

**What are some example queries that trigger this Instant Answer?**
duckduckhack

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
geeks

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Which existing Instant Answers will this one supercede/overlap with?**
f

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screenshot from 2015-01-04 11 34 28

## Checklist
Please place an 'X' where appropriate.

```
[] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Ubuntu 14.04
        [X] Firefox
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/sagarhani.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+name "IsAwesome sagarhani";
+description "My first Goodie, it let's the world know that sagarhani is awesome";
+primary_example_queries "duckduckhack sagarhani";
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/sagarhani.pm";
+attribution github => ["https://github.com/sagarhani", "sagarhani"],
+
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/sagarhani.pm]: can you delete this empty line here?

File: t/IsAwesome/sagarhani.t [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+zci answer_type => "is_awesome_sagarhani";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw(
+        DDG::Goodie::IsAwesome::sagarhani
+    )],
+    'duckduckhack sagarhani' => test_zci('sagarhani is awesome and has successfully completed the DuckDuckHack Goodie tutorial!'),
+    'duckduckhack sagarhani' => undef,
```
Comment (mintsoft) [inline t/IsAwesome/sagarhani.t]: This should be
```perl
    'duckduckhack ssagarhani is awesome' => undef
```

**Comment by mintsoft** [comment]
@sagarhani Thanks for the clarification; I've put a a few tweaks and fixes against the diff. Once they're addressed it'll be ready for merging

**Comment by sagarhani** [comment]
@mintsoft I have fixed all the mistakes. Do I need to send the Pull Request again ?

**Comment by mintsoft** [comment]
@sagarhani Excellent. You don't need to send the PR again, as you add changes onto the master branch on your fork, they become part of this PR.

**Comment by mintsoft** [comment]
Congratulations on your first goodie @sagarhani :cake:

**Comment by moollaza** [comment]
@sagarhani it's live! https://duckduckgo.com/?q=duckduckhack+sagarhani&t=ffab&ia=answer

**Comment by moollaza** [comment]
Now that you've finished your first IA, we really hope you'll work on another!

**Pull request merged**